web/template/watch.gohtml

```diff
@@ -211,6 +215,41 @@
                         @click="less = !less" x-text="less ? 'show more' : 'show less'">show less
                 </button>
             </div>
+            <!-- Video Sections -->
+            {{ if $stream.IsPast }}
+                <div x-data="{sections: new watch.VideoSectionList({{$stream.ID}})}"
+                     x-init="await sections.fetch()"
```

Comment:
Can the sections be provided in the DOM without requiring an extra api call? Do we really need them in alpine here or could the same be achieved using a plain `range` and sprinkle some alpine on top for the pages?

Reply:
E.g. rendering all of them, use `overflow-x-scroll` and create buttons to scroll the container

Reply:
Yup, good idea. I moved the section to the right of the description with y-scroll. Looks even better imo and not a single line of alpine needed for pagination.